### PR TITLE
adds insights data

### LINKED ISSUE #1: Adding mockdata for insights
Need to add mockdata for insights. These insights should have the following information:

- title (text)
- url (text)
- type of document (text)
- author (text input)
- resources (boolean)
- active = enable/disable (true)
- creator_id (number)
- org_id  (number)
- date (postgres date format)

If an insight has a tag or tags attached to it, they will be stored in the tags_insights table.

### PR DESCRIPTION
**Pull request by mk4111**
Closes #1 

Adds mock data for insights and adds tags to it too.

### CODE REVIEW

**Comment by mk4111** [comment]
Hi @SimonLab - I think the insights data should look like this - could you please have a look and confirm? Thank you!

**Comment by iteles** [comment]
@mk4111 Had a quick glance at this and I believe you're missing a time stamp for the date created?

**Comment by mk4111** [comment]
@iteles, I thought it would be better if the date gets inserted by postgres. We do this in the insights plugin [on creating the insights table](https://github.com/postgres-plugin/insights/commit/937699da56f2b74e7d78fa063c28302a13ce2e7e#diff-cc9cdf9e0f3ed86f0e72a937f682b4d0R10).

**Comment by jay-meister** [comment]
@SimonLab branches are a bit screwed up, i think that is my fault. `initial` is currently our `master` branch..

File: index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var data = {}
+
+data.categories = require('./categories.json');
+data.challenges = require('./challenges.json');
+data.organisations = require('./organisations.json');
+data.people = require('./people.json');
+data.tags_challenges = require('./tags_challenges.json');
+data.tags_organisations = require('./tags_organisations.json');
+data.tags = require('./tags.json');
+
```
Comment (jay-meister) [inline index.js:12]: doesn't insights need to be required in?

**Comment by jay-meister** [comment]
lgtm but i dont see how insights data is ever required in

**Comment by mk4111** [comment]
@JMurphyWeb - yes it needs to be required in! Sorry I didn't know if that was something that npm did automatically or if we needed to, but now that I know that `initial` branch is our `master` it makes sense! Please do require it in!

**Comment by jay-meister** [comment]
@SimonLab over to you

**Review by SimonLab — APPROVED** [review]

**Pull request merged**